execution/wasm/contract.go

```diff
@@ -220,13 +222,126 @@ func (ctx *context) ResolveFunc(module, field string) lifeExec.FunctionImport {
 	}
 
 	switch field {
+	case "create":
+		return func(vm *lifeExec.VirtualMachine) int64 {
+			valuePtr := int(uint32(vm.GetCurrentFrame().Locals[0]))
+			dataPtr := uint32(vm.GetCurrentFrame().Locals[1])
+			dataLen := uint32(vm.GetCurrentFrame().Locals[2])
+			resultPtr := uint32(vm.GetCurrentFrame().Locals[3])
+
+			// TODO: is this guaranteed to be okay? Should be avoid panic here if out of bounds?
+			value := bin.BigIntFromLittleEndianBytes(vm.Memory[valuePtr : valuePtr+ValueByteSize])
+
+			var data []byte
+			copy(data, vm.Memory[dataPtr:dataPtr+dataLen])
+
+			ctx.sequence++
+			nonce := make([]byte, txs.HashLength+8)
```

Comment:
The sequence of the account should be updated too

Reply:
I think if my understanding is correct, I don't think we need to update the account sequence (since it should be handled outside the wasm call) after execution sequence number modification